daisy_workflows/build-publish/windows_container/windows-2019-for-containers-uefi.publish

```diff
@@ -0,0 +1,57 @@
+{{/*
```

Comment:
i think this filename is incorrect

Reply:
Thanks for catching that. Updating filename to be .publish.json.